andela/merry-ah-frontend
# #162727411 Setup redux

#### What does this PR do?
Setup redux
#### Description of Task to be completed?
- Setup the store
- Setup the actions
- Setup the reducer
- Setup the redux devtools
#### How should this be manually tested?
- Run `npm i`
- Run `npm start`
- Check Redux devtools for the intial state of the app
#### Any background context you want to provide?
Redux devtools extension for chrome needs to be installed
#### What are the relevant pivotal tracker stories?
[162727411](https://www.pivotaltracker.com/story/show/162727411)
